src/Codeception/Command/Run.php

```diff
@@ -481,13 +477,13 @@ protected function matchSingleTest($suite, $config): ?array
             // Run single test from working directory
             $realTestDir = realpath(Configuration::testsDir());
             $cwd = getcwd();
-            if (strpos($realTestDir, $cwd) === 0) {
+            if (strpos($realTestDir, (string) $cwd) === 0) {
                 $file = $suite;
                 if (strpos($file, ':') !== false) {
                     [$file] = explode(':', $suite, -1);
                 }
                 $realPath = $cwd . DIRECTORY_SEPARATOR . $file;
-                if (file_exists($realPath) && strpos($realPath, $realTestDir) === 0) {
+                if (file_exists($realPath) && strpos($realPath, (string) $realTestDir) === 0) {
```

Comment:
`$cwd` and `$realTestDir` are used as strings in other places too, I think that it would be better to do casting on assignment.